lib/util/game_setup.rb

```diff
@@ -87,15 +87,15 @@ def set_order(player1, player2)
   end
```

Comment:
seems like the kind of thing we might want to test

Reply:
Added this [test](https://github.com/saralein/tic-tac-toe_ruby/pull/23/files#diff-affc662720a70899519e4bd6f5bd03e0R101) for the prompt.